docs/scripts/build-api-docs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!env bash
```

Comment:
This doesn't actually work for my system.

```
$ yarn build-api-docs
command not found: ./scripts/build-api-docs.sh
```

But `/usr/bin/env` works...

```suggestion
#!/usr/bin/env bash
```

Reply:
:rofl: will do that